docs/containers.conf.5.md

```diff
@@ -405,7 +405,7 @@ if you want to set environment variables for the container.
 
 Define where event logs will be stored, when events_logger is "file".
 
-**events_logfile_max_size**=0
+**events_logfile_max_size**="1Mb"
```

Comment:
1Mb different then 1mb?  I am not used to seeing this capitalized.

Reply:
Can you add a line like:
`A  unit  can  be  b  (bytes),  k  (kilobytes),  m(megabytes), or g (gigabytes).` to the man page.

Reply:
I'd second the thought on the "1Mb" format.  That seems strange.  Unless there's a hard technical reason to use it, I'd convert to "1m" as we use [elsewhere](https://docs.podman.io/en/latest/markdown/podman-run.1.html#memory-m-number-unit)